R/LT-FH_gibbs.R

```diff
@@ -88,7 +86,7 @@ estimate_gen_liability_ltfh = function(h2,
                      }
                    )
       
-      cov = get_cov(h2 = h2, n_sib = ifelse(is.na(cur_nsib), 0, cur_nsib))
+      cov = construct_covmat()
```

Comment:
Is it normal that there is no param anymore?

Reply:
The function estimate_gen_liability_ltfh needs to be changed completely, as the new function construct_covmat takes different arguments than get_cov. I will look at that together with Emil in the next couple of days. For now, I have simply removed all arguments, as this only results in errors